src/utils/tcr.js

```diff
@@ -0,0 +1,99 @@
+const executePending = async ({
+  batchedSend,
+  fromBlock,
+  tcrContract,
+  toBN,
+  type
+}) => {
+  const statusChangeEvents = await tcrContract.getPastEvents(
+    `${type}StatusChange`,
+    { fromBlock }
+  )
+
+  // Take most recent event for each item.
+  const latestStatusChanges = {}
+  statusChangeEvents.forEach(event => {
+    const itemKey = event.raw.topics[3]
```

Comment:
why not use `returnValues`?

Reply:
For some reason, `returnValues` is doesn't have the correct values.
```
{ '0': '0x0cAc752dd80D8951166c558440209D33B1F4Ae68',
     '1': '0x7556F79D01Bd9e56b5fB39804A4dE4EB740Aa7Ee',
     '2': '0x0000000000000000000000000000000000000000000000000000000000000000',
     '3': '3',
     '4': false,
     '5': false,
     _requester: '0x0cAc752dd80D8951166c558440209D33B1F4Ae68',
     _challenger: '0x7556F79D01Bd9e56b5fB39804A4dE4EB740Aa7Ee',
     _tokenID: '0x0000000000000000000000000000000000000000000000000000000000000000', <-----
     _status: '3',
     _disputed: false,
     _appealed: false },
  event: 'TokenStatusChange',
  signature: '0x1d939a93a378cb71bc38b0480cac752dd80d8951166c558440209d33b1f4ae68',
  raw: 
   { data: '0x000000000000000000000000000000000000000000000000000000000000000300000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000',
     topics: 
      [ '0x1d939a93a378cb71bc38b0480cac752dd80d8951166c558440209d33b1f4ae68',
        '0x0000000000000000000000007556f79d01bd9e56b5fb39804a4de4eb740aa7ee',
        '0x0000000000000000000000000000000000000000000000000000000000000000',
        '0x41eb9b665c9d6cc73e97587f7f52165678145c0beebb6376157b06d45fdb9f7d' ] } } <---- 

```

Notice how the `_tokenID` is `0` when it should be `0x41eb9...`.

Reply:
That is strange. I assume it is a web3 bug. Events have been pretty unstable in a lot of versions of web3 1.0. Ok this is fine